### PR TITLE
Added Delete account features to Account.php, added persistent hash table

### PR DESCRIPTION
**Pull request by 3jackdaws**

### CODE REVIEW

**Comment by 3jackdaws** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;SYY7SLZH&#x2F;15-persistent-storage-class-for-key-value-pairs-orders-based-on-name"> Persistent storage class for key value pairs, orders based on name</a>

**Comment by 3jackdaws** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;N1RhIlAc&#x2F;14-delete-own-account"> Delete own Account</a>

File: docs/get_wiki.php [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 set_include_path(realpath($_SERVER['DOCUMENT_ROOT']) . '/assets/php');
 require_once 'Parsedown.php';
+sleep(0);
```
Comment (Ritirbase) [inline docs/get_wiki.php]: redundant

File: assets/php/Account.php [hunk 2]
```diff
@@ -71,6 +71,23 @@ public function getPasswordHash(){
         return $this->user['passwd'];
     }
 
+    public function removeFromDatabase(){
+        sleep(1);
+        if(!$this->user['username']){
```
Comment (Ritirbase) [inline assets/php/Account.php]: should not have a not operator

**Pull request merged**